pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -244,11 +246,65 @@ func createACLConfiglet(ctx *zedrouterContext, aclArgs types.AppNetworkACLArgs,
 	if err != nil {
 		return rules, depend, err
 	}
+	clearUDPFlows(aclArgs, ACLs)
 	rules = append(rules, dropRules...)
 	rules, err = applyACLRules(aclArgs, rules)
```

Comment:
Is there a race where a flow can be added before the new ACL is applied and after you cleared it?
For instance is a UDP packet for the port number arrives in that small window?
Would it work to clear after you apply?